ledgerwallet/manifest.py

```diff
@@ -123,19 +123,27 @@ def get_application_flags(self) -> int:
         else:
             return int(self.json["flags"], 16)
 
+    def get_api_level(self) -> int:
+        return int(self.json["apiLevel"], 10)
+
     def get_binary(self) -> str:
         return os.path.join(self.path, self.json["binary"])
 
     def get_target_id(self) -> int:
         return int(self.json["targetId"], 16)
 
+    def has_api_level(self) -> bool:
+        return self.json.get("apiLevel") is not None
+
     def serialize_parameters(self) -> bytes:
         parameters = []
         for entry, value in self.json.items():
             if entry == "name":
                 parameters.append({"type_": "BOLOS_TAG_APPNAME", "value": value})
             elif entry == "version":
                 parameters.append({"type_": "BOLOS_TAG_APPVERSION", "value": value})
+            elif entry == "apiLevel":
```

Comment:
According to https://github.com/LedgerHQ/blue-loader-python/commit/c7b1f9ee2be149b1dcf4a03d235137ee0a9e627a, API level is not a serialized parameter.